onnxmltools/convert/coreml/datatype.py

```diff
@@ -42,31 +42,31 @@ def _convert(cm_type):
     return COREML_TYPE_TO_ONNX_TYPE[cm_type]
 
 
-def _handle_scalar_feature(cm_value):
+def _handle_scalar_feature(cm_value, doc_string=''):
     which_type = cm_value.type.WhichOneof('Type')
     onnx_type = _convert(which_type)
     onnx_shape = [1]
-    return model_util.make_tensor_value_info(cm_value.name, onnx_type, onnx_shape)
+    return model_util.make_tensor_value_info(cm_value.name, onnx_type, onnx_shape, doc_string)
 
 
-def _handle_multi_array_feature(cm_value, batch_size=1):
+def _handle_multi_array_feature(cm_value, batch_size=1, doc_string=''):
     data_type = cm_value.type.multiArrayType.dataType
     onnx_type = _convert(data_type)
     onnx_shape = [batch_size]
     for shape_val in cm_value.type.multiArrayType.shape:
         onnx_shape.append(shape_val)
-    return model_util.make_tensor_value_info(cm_value.name, onnx_type, onnx_shape)
+    return model_util.make_tensor_value_info(cm_value.name, onnx_type, onnx_shape, doc_string)
 
 
-def _handle_dictionary_feature(cm_value):
+def _handle_dictionary_feature(cm_value, doc_string=''):
     key_type = cm_value.type.dictionaryType.WhichOneof('KeyType')
     onnx_key_type = _convert(key_type)
     onnx_value_type = onnx_proto.TensorProto.FLOAT
-    map_type = model_util.make_map_value_info(cm_value.name, onnx_key_type, onnx_value_type)
+    map_type = model_util.make_map_value_info(cm_value.name, onnx_key_type, onnx_value_type, doc_string)
     return map_type
 
 
-def _handle_image_feature(cm_value, batch_size=1):
+def _handle_image_feature(cm_value, batch_size=1, doc_string=''):
```

Comment:
This function should not include doc_string as its argument since doc_string has been used to describe the tensor format of image type. Another possibility is to concatenate our string and the short description from CoreML.

Reply:
Adding concatenate.